lib/http.js

```diff
@@ -450,6 +451,37 @@ async function _authorize({
   };
 }
 
+/**
+ * Gets an expectedRootCapability from an invocationTarget.
+ *
+ * @param {object} options - Options to use.
+ * @param {string | Array<string>} options.invocationTarget - An
+ *   invocationTarget.
+ * @param {string} options.keystoreId - A url to a key store.
+ *
+ * @return {string | Array<string>} The expectedRootCapability for
+ *   the invocationTarget.
+ */
+function _getExpectedRootCapability({invocationTarget, keystoreId}) {
+  // while no invocationTargets in this module are arrays
+  // it is possible for an invocationTarget to be an array.
+  if(Array.isArray(invocationTarget)) {
+    return invocationTarget;
+  }
+  // all code below assumes invocationTarget is a string.
+  assert.string(invocationTarget, 'invocationTarget');
```

Comment:
It's fine to do this check ... but this should be caught before we ever this this helper because we don't want assertion errors to be returned to the client.

Reply:
Removed it here: https://github.com/digitalbazaar/bedrock-kms-http/pull/14/commits/8b5eb53e0fe3780dcea82786dab1571f7f0b083b